### DISCUSSION TITLE
Can't Arm/Disarm after trial subscription ended

### DISCUSSION THREAD

**Discussion by plimps06** [tsightler/ring-mqtt#895] [Support Q&A]
Hi all,
I've been using Ring-MQTT (no Cameras) with HA and HomekitBridge quite seamlessly for the last month. My trial subscription to Ring expired yesterday and I can no longer Arm/Disarm the alarm system in HA or Homekit. I still get notifications when I manually Arm/Disarm. I tried getting a new token, I uninstalled Ring-MQTT and reinstalled with a new token. No luck. Using HA Green and Mosquitto broker. Updated to the newest versions of all. I have been getting an error in the logs:
Error: Failed to set alarm mode to "some".  Sensors may require bypass, which can only be done in the Ring app.

However, I have no sensors that are open and need bypassing.

**Comment by tsightler** [comment]
Subscriptions are required to arm/disarm unless you purchased your alarm prior to March 29th, 2023 or something like that.  Google will know the exact date.

**Reply by plimps06** [reply]
Seems like this should be spelled out more clearly in the Read Me, wouldn't have bought a Ring system if I had known this. Pretty clearly lists the feature of arming/disarming in the notes without clarifying that it requires a subscription (as it does for other features).

**Reply by tsightler** [reply]
I don't control what Ring decides to charge for or does not, nor do I spend my life reading Ring announcements to figure it out.  This is just a personal project I was dumb enough to share on the Internet, no one should make purchasing decisions based on that.  I never make any recommendation to purchase a Ring product, and, actually, I actively discourage users that don't want to pay for subscriptions to buy something else.

Ring is actually quite clear about their requirements, here's a quote directly from the Ring Alarm webpage:

> Ring Alarm and all Ring Alarm accessories require a subscription for in-app features and digital notifications, including **digital arming / disarming** and integration with other Ring, Echo, Alexa, and third party products. Subscription sold separately. View [ring.com/protect-plans](https://ring.com/protect-plans) for pricing and details. Customer reviews may reference features that require a subscription.
